Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -2945,7 +2984,14 @@ Tests customizing continue button ("some of the time").
         [items addObject:@"Item2"];
         [items addObject:@"Item3"];
         stepViewController.navigationItem.titleView = [[UISegmentedControl alloc] initWithItems:items];
+    }else if ([stepViewController.step.identifier isEqualToString:@"waitTask.step2"]) {
+        // Indeterminate step
+        [((ORKWaitStepViewController *)stepViewController) performSelector:@selector(finish) withObject:nil afterDelay:5.0];
```

Comment:
This is reference back to comment above about changing from `ORKActiveStep` to `ORKStep`.